pom.xml

```diff
@@ -153,12 +153,6 @@
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
-            <artifactId>pipeline-stage-step</artifactId>
```

Comment:
Actually running the example requires all sorts of plugins which `workflow-cps` does not depend on. If we could define this snippets in another plugin like `workflow-aggregator`, I would do that, but @tfennelly did not write such an extension point.